content/releases/3-0-1.md

```diff
@@ -0,0 +1,124 @@
+- [Message events](http://localhost:1313/how-to/bpmn/bpmn-elements/#events) now support topics that use the `+` wildcard.
+- Better support for [Work calendars]({{< relref "/how-to/work-calendars" >}})
+  - When configured, calendar data persists to a timeseries DB.
+  - `Breaking` Work calendars now query for `WorkCalendarDefinitions` not `workCalendars`.
+  This makes it possible to configure calendars in the UI.
+  - Work calendar execution is now based on Hierarchy Scope.
+
+**Fixes**
+- Reduce number of retries in [BPMN recovery]({{< relref "/deploy/maintain/bpmn-nodes/" >}}), speeding up response times when node does not exist.
+ For the user's experience, the delay was particularly notable on [API BPMN triggers]({{< relref "/how-to/bpmn/trigger-workflows/#start-a-workflow-from-an-api" >}}) when the specified workflow did not exist.
+- Capture panics in JSONata go for better recovery and discovery.
+- `Breaking`. Limit BPMN users to only tokens with specific audience and scope.
+- Fall back to looking up specs in the database. Fixes issue when spinning up new development containers.
+- Include metadata fields for `jobResponse`. Fixes panic if node tried to access these fields.
+- Respect [`debug`](/how-to/bpmn/debug-workflows/#adding-the-debug-flag) flag for BPMNs triggered with `createAndRunBpmn`.
+- Reuse expected Audience for clients to match documented client name.
+- Do not create calendar entry for end events.
+- Allow `__traceDebug` to be set from [variable context]({{< relref "/how-to/bpmn/create-workflow/#access-process-variable-context" >}}).
+
+
+### Schema
+
+**Features**
+- `breaking`. Add `id` and `label` to `stateTransitionInstance`.
+- Add menu URL for portal in UI (preparation for future work in UI).
+- Add filter to query material actual by a material requirement and vice-versa.
+- Link `workCalendarEntries` to equipment versions.
+- Include Kafka in the enum values for data sources.
+- Added optional `InformationObjectTypeEnum` to `OperationsEventRecordEntry`.
+
+**Fixes**
+- Add 0-n relationship from `materialRequirements` to `materialActual`.
+- The `segmentResponse` entity can now have multiple `jobResponses` attached.
+- `Breaking`. Change `stateModelTransition` from `state` to a relationship.
+- link `operationsEventDefinitionProperty` to `operationsEventProperty`.
+
+### Core
+
+**Features**
+- Add validation to config files.
+- Obfuscate sensitive information when published.
+- Create operations event on migrations.
+- Create `AppContext` to contain `context.Context`, `trace.Span`, and `logger` that can be passed around functions within the application.
+
+**Fixes**
+- Standardize config casing.
+- Prohibit secrets and environment variables from having the same ID. This fixes a bug in BPMN execution.
+- Fix `saveVersionAs` bug.
+
+**Refactor**
+- Change config file from JSON to YAML.
```

Comment:
The above line is a breaking change, throw it up the top, and add:

Change configuration keys to camel case and to consistent between microservices. (Backward compatible)